teamserver/teamserver/events/worker.py

```diff
@@ -42,7 +45,9 @@ def notify_integration(integration, data):
     Trigger an integration with the given data.
     """
     if integration and isinstance(integration, Integration):
-        integration.run(data)
+        # Check if the integration is enabled
+        if integration.config.get("enabled", False):
+            integration.run(data)
```

Comment:
Make sure to call notify_integration.delay() for your integration at the bottom of this file. See the slack example. Line 65/70